src/cli/cmds/clone/build-blueprint-commands.js

```diff
@@ -0,0 +1,26 @@
+import _merge from 'lodash/merge';
+import _cloneDeep from 'lodash/cloneDeep';
+
+import getEnvironment from '../../environment';
+import Clone from '../../../sub-commands/clone';
+import buildBlueprintCommand from './build-blueprint-command';
+
+const loadBlueprintSettings = (blueprint, bp) =>
+  (blueprint.settings = _merge(
+    _cloneDeep(bp.common),
+    _cloneDeep(bp[blueprint.name])
+  ));
+
+const buildBlueprintCommands = () => {
```

Comment:
All of the blueprints will already be found, read,  and have it's settings already merged.  Again the api is not quite ready yet, but soon.  And any suggestions would be most welcome. see #132 for current thinking